src/main/java/org/apache/commons/lang3/builder/HashCodeBuilder.java

```diff
@@ -37,7 +37,7 @@
  *
  * <p>
  * This class enables a good {@code hashCode} method to be built for any class. It follows the rules laid out in
- * the book <a href="http://www.oracle.com/technetwork/java/effectivejava-136174.html">Effective Java</a> by Joshua Bloch. Writing a
+ * the book <a href="https://www.oracle.com/technetwork/java/effectivejava-136174.html">Effective Java</a> by Joshua Bloch. Writing a
```

Comment:
There is no such page, it redirects to https://www.oracle.com/java/technologies/effectivejava.html

Reply:
URL replaced by https://www.oracle.com/java/technologies/effectivejava.html.

Reply:
Did you change the URL? I still see the old one.